packages/gestalt/src/contexts/ColorScheme.js

```diff
@@ -35,34 +45,54 @@ const lightModeTheme = {
   name: 'lightMode',
   colorRed0: '#ff5247',
   colorRed100: '#e60023',
+  colorRed100Active: '#a3081a',
+  colorRed100Hovered: '#ad081b',
   colorGray0: '#fff',
+  colorGray0Active: '#e0e0e0',
+  colorGray0Hovered: '#f0f0f0',
   colorGray50: '#fff',
   colorGray100: '#efefef',
+  colorGray100Active: '#dadada',
+  colorGray100Hovered: '#e2e2e2',
   colorGray150: '#ddd',
   colorGray200: '#767676',
+  colorGray200Active: '#828282',
+  colorGray200Hovered: '#878787',
   colorGray300: '#111',
   colorGray400: '#000',
   colorTransparentDarkGray: 'rgba(51, 51, 51, 0.8)',
   colorTransparentGray60: 'rgba(0, 0, 0, 0.06)',
   colorTransparentGray100: 'rgba(0, 0, 0, 0.1)',
   colorTransparentWhite: 'rgba(255, 255, 255, 0.8)',
+  blueHovered: '#4a8ad4',
+  blueActive: '#4a85c9',
 };
 
 const darkModeTheme = {
   name: 'darkMode',
   colorRed0: '#e60023',
   colorRed100: '#ff5247',
+  colorRed100Active: '#a3081a',
+  colorRed100Hovered: '#ad081b',
   colorGray0: '#030303',
+  colorGray0Active: '#1f1f1f',
+  colorGray0Hovered: '#121212',
   colorGray50: '#212121',
   colorGray100: '#404040',
+  colorGray100Active: '#505050',
+  colorGray100Hovered: '#494949',
   colorGray150: '#585858',
   colorGray200: '#ababab',
+  colorGray200Active: '#9b9b9b',
+  colorGray200Hovered: '#919191',
   colorGray300: '#efefef',
   colorGray400: '#fff',
   colorTransparentDarkGray: 'rgba(255, 255, 255, 0.8)',
   colorTransparentGray60: 'rgba(250, 250, 250, 0.5)',
-  colorTransparentGray100: 'rgba(0, 0, 0, 0.5)',
+  colorTransparentGray100: 'rgba(250, 250, 250, 0.6)',
```

Comment:
I noticed this color looked bad during testing and I think it's not in the official palette anyway